static/app/views/starfish/views/webServiceView/spanGroupBar.tsx

```diff
@@ -21,10 +25,16 @@ function getPercent(value, total) {
   return Math.round((value['sum(span.self_time)'] / total) * 100 * 100) / 100;
 }
 
-export function SpanGroupBar() {
+type SpanGroupBarProps = {
+  onHover: (string) => void;
```

Comment:
```suggestion
  onHover: (string: string | null) => void;
```
seems to be missing a type here